File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.cpp
class Solution {
public:
    int minBitFlips(int start, int goal) {
        int set_missing_bits = start^goal;
        int cnt=0;
        while(set_missing_bits){
            cnt+=set_missing_bits&1;
            set_missing_bits>>=1;
        }
        return cnt;
    }
};

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.py
class Solution:
    def minBitFlips(self, start: int, goal: int) -> int:
        # 10 - 1010
        # 7  - 0111
        res = start ^ goal 
        cnt = 0
        while(res):
            cnt+=res&1 
            res>>=1 
        return cnt 